### ISSUE TITLE
CSS function returns [object Object] in Handlebars

### ISSUE THREAD

**Issue by smth** [carrot/roots-css-pipeline#21]
When attempting to use `{{ CSS }}` in a Handlebars template it is returning `<link rel='stylesheet' href='[object Object]css/styles.css' />`. This does not happen using the same config (below) with Jade.

``` coffee
extensions: [
  css_pipeline(files: ['assets/css/vendor/**', 'assets/css/main.scss'], out: 'css/styles.css')
]
```

**Comment by jescalan** [comment]
Strange. `css` is a function, is it being executed by handlebars?

**Comment by smth** [comment]
OK, I'm not sure.. I put `{{ CSS }}` in the Handlebars template. In place of `!= css()` in the Jade one.

**Comment by jescalan** [comment]
So I'm not sure how case is handled, but the function is `css` in lower case. It's also a function so it does need to be called. I might try `{{ css() }}`

**Comment by smth** [comment]
Yeah, I was using lowercase (typo above). I did also try `{{ css() }}` but that resulted in:

```
Expecting 'ID', 'STRING', 'NUMBER', 'BOOLEAN', 'UNDEFINED', 'NULL', 'DATA', got 'INVALID'
```

**Comment by jescalan** [comment]
Weird. Well I must admit, I do not really use handlebars ever so I don't know how they handle functions. As long as it calls the function it will output the correct string though, there's nothing else I can do to fix this really...

**Comment by smth** [comment]
This is my first time using Handlebars, so don't know much about it either. I just figured there might be a bug there, as it gets close to outputting correctly, i.e it gets `<link rel='stylesheet' href='` and `css/styles.css' />`. It's just the `[object Object]` that shouldn't be there.

**Comment by jescalan** [comment]
If it works in jade, it's probably not a bug in the core code... it's just a function that when executed returns a string

**Comment by jescalan** [comment]
If you want to dig a little deeper though you are welcome to! The function is [here](https://github.com/carrot/roots-css-pipeline/blob/master/lib/index.coffee#L45-L54), you'd just have to jump in and run some `console.logs`, tinker around with it a bit 😁

**Comment by guill3m** [comment]
Ran into the same problem today. Solved it by doing `{{{css '/'}}}` instead, if it helps someone.
